javascript/javascript1/week2/Event-applicatoin.js

```diff
@@ -1,2 +1,11 @@
-weekday [ 'monday', 'tuesday', 'wednesday', 'thursday',
-'friday', 'saturday', 'monday' ];
\ No newline at end of file
+const weekDays = ['monday', 'tuesday', 'thursday', 'wednesday',
+ 'thursday', 'friday', 'saturday', 'sunday'];
+const today = new Date();
+const day = today.getDay()
+let numDaysToEvent;
+function getEventWeekDay (numDaysToEvent) {
+ const dayOfTheEvent =( day + numDaysToEvent) % 7 ;
```

Comment:
Variable name convention should be related, here this variable gives a number instead of a day.